app/webpacker/scripts/global/disable-browser-autofill.js

```diff
@@ -0,0 +1,23 @@
+function disableBrowserAutofill () {
+  const dataAttributeName = 'data-nameoriginal'
+  const inputTypes = ['text', 'tel', 'email', 'number']
+
+  document.querySelectorAll('form').forEach(form => {
+    if (form.getAttribute('autocomplete') === 'off') {
+      const inputs = [...form.querySelectorAll('input[autocomplete]')].filter(e => inputTypes.includes(e.type))
```

Comment:
filter creates a new array so you don't have to use spread `...`. 

conventionally `e =>` reads as "event" (or error). I would suggest changing e for readability if anything 

```suggestion
      const inputs = form.querySelectorAll('input[autocomplete]').filter(input => inputTypes.includes(input.type))
```

Reply:
`form.querySelectorAll()` doesn't have the `filter` method, that's why I went for the spread option.

You're right about `e` though, it's a common understanding that it usually represents an event. I was thinking of element at the time - will change it.